libs/common-ui/src/lib/atoms/Input/Input.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Input } from './Input';
+
+const meta = {
+  title: 'UI/Atoms/Input',
```

Comment:
good name! But rethink in terms of accessibility if input needs a label. If so, it may become molecule 🤔